docs/relying-parties/reference/metrics-for-relying-parties.md

```diff
@@ -4,7 +4,38 @@ title: Metrics for Relying Parties
 sidebar_label: Metrics
 ---
 
-Last updated: `January 26th, 2022`
+Last updated: `March 20th, 2023`
+
+## Subscriptions Services Data Sources
+Subscription Platform provides product metrics by obtaining subscriptions data from the following sources: Stripe, mobile app store providers, and FxA logs.
+
+### Stripe
+The bulk of Subscriptions Services data comes from Stripe and is synced to BigQuery using Fivetran and its [Stripe connector](https://fivetran.com/docs/applications/stripe). This data reflects the current subscription state of all active subscriptions paid for via Stripe or PayPal. It also includes information regarding subscriptions and customers, as well as products, plans, invoices, cards, charges, discounts, coupons, and promotion codes.
+
+Some of the customer information includes the customer's postal code, state, and country for a subscription, which helps resolve issues for customers across a particular dimension (e.g., country, payment provider, etc.). However, as it intentionally excludes any personal identifiable information of customers (e.g., name, street address, phone number, email address), it can be challenging to use to help on an individual customer level. Other uses for this data is for retrieving all-time counts (e.g., total number of active subscriptions).
+
+Fivetran preserves historical data within `subscription_history` by comparing the current state of the subscription with the most recent record in the history table and adding a new record if they vary. However, there are limitations with the Fivetran Stripe sync, as it only records historical state/changes for top-level subscription fields and not the associated subscription items which indicate the subscribed plan. As we have to rely on the subscription metadata fields `previous_plan_id` and `plan_change_date`, which only records the most recent plan change, we can potentially miss/lose history if there are any problems with the Fivetran sync.
+
+As of February 2023, Subscription Platform also syncs Stripe Firestore Customer and Subscriptions collections.
+
+### Mobile app store providers
+Subscriptions Services data is also obtained from two mobile app store providers: Apple and Google. This includes date of billing and expired dates, subscription change event, etc. Apple provides limited data - it does not provide the country, price, or currency of subscriptions.
+
+As of December 2022, Subscription Platform supports [Apple IAP][apple-iap-integration] subscriptions, and stores both current and historical subscription state from that point forward in the Firestore database and synced in BigQuery.
```

Comment:
s/current and historical/current? You have a paragraph further down that covers historical state.